controllers/managedmcg_controller.go

```diff
@@ -594,3 +610,38 @@ func (r *ManagedMCGReconciler) getCSVByPrefix(name string) (*opv1a1.ClusterServi
 
 	return csv, nil
 }
+
+func (r *ManagedMCGReconciler) ensureConsolePlugin(clusterVersion string) error {
+	// The base path to where the request are sent
+	basePath := console.GetBasePath(clusterVersion)
+
+	// Get mcg console Deployment
+	mcgConsoleDeployment := console.GetDeployment(r.namespace)
+	err := r.Client.Get(context.TODO(), types.NamespacedName{
```

Comment:
```suggestion
	err := r.Client.Get(r.ctx, types.NamespacedName{
```
Please update all such instances.

Reply:
IMO, it is always better to fetch the ctx from methods that saved one. I will change it suggested way to keep this things matched